### PR TITLE
fix: Fixed Ui error in User Profile

### LINKED ISSUE #553: Improper UI to show error  on User profile in no data.
**Summary:**

UI is showing partial data and on top of it render error .

**Steps to reproduce:**

Open the app then open the drawer then click on the profile .

**Expected behavior:**

If it is showing error then it shouldn't show any data and the error message should occupy the whole fragment.

**Observed behavior:**
On the UI some partial data was showing and on top of it, the error message was being displayed.

**Device and Android version:**
Device: Moto G-3.
Android version: 6.0.1.

**Screenshots:**
screenshot_20180202-004340
I would like to work on this issue.

### PR DESCRIPTION
**Pull request by ivary43**
Fixes #553

Please Add Screenshots If any UI changes.

Please make sure these boxes are checked before submitting your pull request - thanks!

- \[x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- \[x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- \[x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by ivary43** [comment]
@dilpreet96 , @therajanmaurya  please review.

**Comment by roychowdhuryrohit-dev** [comment]
Please merge this issue. @dilpreet96  @therajanmaurya

File: app/src/main/res/layout/fragment_user_profile.xml [hunk 1]
```diff
@@ -48,7 +48,9 @@
         <LinearLayout
             android:layout_height="match_parent"
             android:layout_width="match_parent"
-            android:orientation="vertical">
+            android:orientation="vertical"
+            android:id="@+id/ll_user_profile"
+            android:visibility="visible">
```
Comment (dilpreet2028) [inline app/src/main/res/layout/fragment_user_profile.xml]: @ivary43 I don't thing we need to declare visibility( as visible) as it is the default so can you remove this?

**Review by dilpreet2028 — CHANGES_REQUESTED** [review]

**Comment by ivary43** [comment]
@dilpreet96  done. Please review :)

**Review by dilpreet2028 — APPROVED** [review]

**Comment by dilpreet2028** [comment]
@ivary43 can you change your commit message according to the guidelines : https://github.com/openMF/mifos-mobile/wiki/Commit-Style-Guide#the-type

**Comment by ivary43** [comment]
@dilpreet96 done.

**Comment by dilpreet2028** [comment]
@ivary43 you have changed the title of this PR not the commit message, please change the commit message

**Comment by ivary43** [comment]
@dilpreet96 I have changed the commit message please review.

**Pull request merged**